packages/autocomplete-shared/src/core/AutocompleteOptions.ts

```diff
@@ -177,15 +176,6 @@ export interface AutocompleteOptions<TItem extends BaseItem> {
    * @link https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-js/autocomplete/#param-reshape
    */
   reshape?: Reshape<TItem>;
-  /**
-   * Whether to enable the Insights plugin and load the Insights library if it has not been loaded yet.
-   *
-   * See [**autocomplete-plugin-algolia-insights**](https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-plugin-algolia-insights/) for more information.
-   *
-   * @default false
-   * @link https://www.algolia.com/doc/ui-libraries/autocomplete/api-reference/autocomplete-js/autocomplete/#param-insights
-   */
-  insights?: CreateAlgoliaInsightsPluginParams | boolean;
```

Comment:
`insights` option can't be in `autocomplete-shared` package anymore because it would create a circular dependency (the type depends on `CreateAlgoliaInsightsPluginParams` type, wich is part of the `@algolia/autocomplete-plugin-algolia-insights` package).